feat: follow indirect repository URLs (#629)
oracle/macaron: docs/source/pages/using.rst

- ``repo_pom_paths`` (Values: List of POM tags) - Determines where to search for repository information in the POM files. E.g. scm.url.
- ``find_parents`` (Values: True or False) - When enabled, the Repository Finding feature will also search for repository URLs in parents POM files of the current dependency.

Under the related header ``repofinder``, three more options exist: ``find_repos``, ``use_open_source_insights``, and ``redirect_urls``:

- ``find_repos`` (Values: True or False) - Enables or disables the Repository Finding feature.
- ``use_open_source_insights`` (Values: True or False) - Enables or disables use of Google's Open Source Insights API.

> - ``redirect_urls`` (Values: List of URLs) - Found URLs that exist in this list will be returned as the URL they redirect to.

Reviewer: I found this sentence a bit hard to follow. How about:

```suggestion
- ``redirect_urls`` (Values: List of URLs) - These are the URLs that redirect to the actual repository URL.
```
